package-lock.json

```diff
@@ -11,7 +11,9 @@
       "license": "Apache-2.0",
       "dependencies": {
         "js-sha3": "^0.8.0",
+        "lodash": "^4.17.21",
         "typescript": "^4.1.5",
+        "underscore": "^1.13.1",
```

Comment:
If I run `npm i` npm edits the package-lock file, but after that change, it does correctly return clear for `npm audit`. Can you check if your `npm i` triggers any additional changes?

Reply:
Also a rebase should fix all the test issues